crates/miden-lib/asm/miden/auth/rpo_falcon512.masm

```diff
@@ -81,10 +79,11 @@ end
 #! the owner public key mapping - the previous signers must authorize the change to the new signers,
 #! not the new signers authorizing themselves.
 #!
-#! Inputs:  [pub_key_slot_idx, num_of_approvers, MSG]
+#! Inputs:  [pub_key_slot_prefix, pub_key_slot_suffix, num_of_approvers, MSG]
```

Comment:
Question (and probably not for this PR): since now storage slots are named, do we need to pass in the slot index? The alternative being is that we'd import the constant defining the storage slot name where the public key is expected to be.